docker-compose-e2e-readme.md

```diff
@@ -0,0 +1,140 @@
+# Opzetten/Opstarten containerized BRP Bevragen end-to-end test omgeving
+
+Start de containers met behulp van de volgende statement
+
+```
+docker-compose -f docker-compose-e2e.yml up -d
+```
+
+Stop de containers met behulp van de volgende statement
+
+```
+docker-compose -f docker-compose-e2e.yml down
+```
+
+## Beheren van de BRP bevragen test database
+
+Dit kan worden gedaan met behulp van de pgAdmin tool. Deze wordt opgestart door in een browser naar de volgende url te navigeren
+
+```
+http://localhost:5433
+```
+
+login met Email Address / Username: admin@haalcentraal.nl en Password: root
+
+### Connectie maken met de database
+
+rechter muis klik op Servers. Selecteer Register > Server...
+
+![Register Server](./docs/img/pgAdmin-register-server.png)
+
+
+In de Register-Server dialog box, vul de volgende velden in
+
+![Register Server dialog box](./docs/img/pgAdmin-register-server-dialog.png)
+
+password: root
+
+klik op de **Save** button
+
+De volgende stappen zijn alleen nodig als de BRP database nog niet is ge-restored
+
+### Aanmaken Login/Group Role 'haalcentraal'
+
+rechter muis klik op Login/Group Roles. Selecteer Create > Login/Group Role...
+
+![Create Login/Group Role](./docs/img/pgAdmin-create-login-role.png)
+
+In de Create - Login/Group Role dialog box, vul de Name veld en selecteer alle privileges
+
+![Create Login/Group Role dialog box](./docs/img/pgAdmin-create-login-role-dialog.png)
+
+klik op de **Save** button
+
+### Aanmaken 'rvig_haalcentraal_testdata' database
+
+rechter muis klik op Databases. Selecteer Create > Database...
+
+![Create Database](./docs/img/pgAdmin-create-database.png)
+
+In de Create - Database dialog box, vul de volgende velden in
+
+![Create Database dialog box](./docs/img/pgAdmin-create-database-dialog.png)
+
+klik op de **Save** button
+
+### Restore 'rvig_haalcentraal_testdata' database
```

Comment:
volgens mij publiceren we de haal centraal database niet, dus kan niemand dit uitvoeren. Is denk ik alleen interessant voor ons. Moeten we die dan niet juist verwijderen zodat dit niet in master komt? anders gaan afnemers dit ook proberen en lopen (pas) op deze stap vast.

Met andere woorden, moeten deze, en Docker-compose-e2e.yml niet beter worden verwijderd? Wij zelf kunnen die prima lokaal zelf toevoegen en aan gitignore of git/info/exclude toevoegen